src/gpl/src/nesterovBase.cpp

```diff
@@ -1899,6 +2038,24 @@ getOverlapArea(const Bin* bin, const Instance* inst) {
   }
 }
 
+
+// A function that does 2D integration to the density function of a
+// bivariate normal distribution with 0 correlation.
+// Essentially, the function being integrated is the product
+// of 2 1D probability density functions (for x and y). The means and standard deviation of the 
+// probablity density functions are parametarized.
+// In this function, I am using the closed-form solution of the integration.
+// The limits of integration are lx->ux and ly->uy
+static float integrate(integrationParams i)
```

Comment:
integrate is a bit too general a name for this function. I would probably name it something more like calculateBiVariateNormalCDF(...).

I think renaming the struct from integrationParams to biNormalParameters might be helpful as well. That way it could potentially be used for more than just the integration.